src/Microsoft.Fx.Portability.MetadataReader/AssemblyReferenceInformation.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace Microsoft.Fx.Portability.ObjectModel
+{
+    public class AssemblyReferenceInformation
+    {
+        private readonly string _string;
+
+        public AssemblyReferenceInformation(string name, Version version, string culture, string publicKeyToken)
+        {
+            Name = name;
+            Version = version;
+            Culture = culture;
+            PublicKeyToken = publicKeyToken;
+
+            _string = $"{Name}, Version={Version}, Culture={Culture}, PublicKeyToken={PublicKeyToken}";
+        }
+
+        public string Name { get; }
+
+        public string Culture { get; }
+
+        public Version Version { get; }
+
+        public string PublicKeyToken { get; }
+
+        public override string ToString() => _string;
+
+        public override bool Equals(object obj)
+        {
+            return string.Equals(_string, obj.ToString(), StringComparison.OrdinalIgnoreCase);
+        }
+
+        public override int GetHashCode()
+        {
+            return _string.GetHashCode();
```

Comment:
I'm using the StringCOmparison.OrdinalIgnoreCase for the equality and gethashcode now. The file has moved (I moved it to the core library instead of the metadata reader)